### PR TITLE
:chart_with_upwards_trend: Authorized Studies Card GA tracking

### PR DESCRIPTION
**Pull request by bdolly**
Tracks hovers and clicks in GA for the User Dashboard Authoerized Studies card

### CODE REVIEW

File: src/components/UserDashboard/AuthorizedStudies/Gen3Connected.js [hunk 1]
```diff
@@ -95,10 +96,22 @@ const Gen3Connected = ({
   }, {});
 
   const onStudyTotalClick = studyId => () => {
+    debugger;
```
Comment (ciaranschutte) [inline src/components/UserDashboard/AuthorizedStudies/Gen3Connected.js]: @bdolly remove debugger please

File: src/components/UserDashboard/AuthorizedStudies/Study.js [hunk 2]
```diff
@@ -33,12 +33,14 @@ const Codes = styled(StudyCount)`
   margin-bottom: 5px;
 `;
 
+// isn't this an internal link?
```
Comment (ciaranschutte) [inline src/components/UserDashboard/AuthorizedStudies/Study.js]: @bdolly correct. bad semantic use. will fix in different branch, let's leave comments like this to a different channel (slack/email) so it doesn't set a precedent for communication

File: src/components/UserDashboard/AuthorizedStudies/Gen3Connected.js (hunk 1 above)
Comment (bdolly) [reply src/components/UserDashboard/AuthorizedStudies/Gen3Connected.js]: 👍

File: src/components/UserDashboard/AuthorizedStudies/Study.js (hunk 2 above)
Comment (bdolly) [reply src/components/UserDashboard/AuthorizedStudies/Study.js]: @ciaranschutte  understood

**Pull request merged**